types/simplecrawler/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for simplecrawler 1.1
+// Project: https://github.com/simplecrawler/simplecrawler
+// Definitions by: Pedro Mutter <https://github.com/MutterPedro>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import Crawler from './crawler';
+import * as queue from './queue';
+import * as cache from './cache';
+
+export default Crawler;
```

Comment:
It seems like the [underlying library](https://www.npmjs.com/package/simplecrawler#getting-started) is not using `export default`, so this .d.ts file should not either. Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.